openshift/tests-extension/.openshift-tests-extension/openshift_payload_olmv1.json

```diff
@@ -1,4 +1,34 @@
 [
+  {
+    "name": "[sig-olmv1][OCPFeatureGate:NewOLMOwnSingleNamespace][Skipped:Disconnected][Serial] OLMv1 operator installation support for singleNamespace watch mode with quay-operator should install a cluster extension successfully",
```

Comment:
/hold 

We TBD that would not cause issues, add the Serial
+ green flag to merge ( we might wait the release branch cut to do so )